js/ui/form/ui.form.item_options_actions.js

```diff
@@ -1,24 +1,44 @@
 import ItemOptionAction from "./ui.form.item_option_action";
 import { data } from "../../core/element_data";
+import { extend } from "../../core/utils/extend";
+import { getFullOptionName } from "./ui.form.utils";
 
 class WidgetOptionItemOptionAction extends ItemOptionAction {
     tryExecute() {
+        const { value } = this._options;
         const instance = this.getInstance();
-        instance.option(this.value);
+        instance.option(value);
         return super.tryExecute();
     }
 }
 
+class TabOptionItemOptionAction extends ItemOptionAction {
+    tryExecute() {
+        const { optionName, value, previousValue, item } = this._options;
+        const instance = this.getInstance();
+        const targetedTitle = optionName === "title" ? previousValue : item.title;
+        const tabPanelItems = instance.option("items") || [];
+        const itemIndex = tabPanelItems.map(item => item.title).indexOf(targetedTitle);
```

Comment:
Linking objects by the [title](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/TabbedItem/tabs/#title) string doesn't look safe